### DISCUSSION TITLE
Unable to use methods from sqlalchemy

### DISCUSSION THREAD

**Discussion by abhinay181** [sqlalchemy/sqlalchemy#10501] [Usage Questions]
I am unable to access methods from sqlalchemy package. I have installed the package successfully you can see it in below screenshots. I have run the pip list command to show that sqlalchemy is installed
![new 2](https://github.com/sqlalchemy/sqlalchemy/assets/85485413/ac77df03-af02-4ca0-8463-11b3cf6629d9)
![new 3](https://github.com/sqlalchemy/sqlalchemy/assets/85485413/c62bfe0d-4fb9-49d8-8e56-5815b69b407d)
![new 4](https://github.com/sqlalchemy/sqlalchemy/assets/85485413/89ff1931-cf2b-44aa-8f71-560257ead11b)

@zzzeek  Can you please help me to resolve this issue? which is blocking my work.

**Comment by abhinay181** [comment]
@zzzeek @CaselIT Can anyone help on this issue

**Comment by zzzeek** [comment]
do you have any other modules called something like "sqlalchemy.py" ?  that could produce a conflict

**Reply by abhinay181** [reply]
@zzzeek 

To verify that I have run pip show sqlalchemy.py in the terminal and it says package not found

PS C:\Users\ajallipa\Desktop\Learning Projects\functions> pip show sqlalchemy.py
WARNING: Package(s) not found: sqlalchemy.py

**Reply by CaselIT** [reply]
`pip show` accepts a package name not a file name
